docs/Operators.md

```diff
@@ -177,14 +177,16 @@ expect(node, inputs=[x], outputs=[y],
   
   If necessary the right-hand-side argument will be broadcasted to match the
   shape of left-hand-side argument. When broadcasting is specified, the second
-  tensor can either be of size 1 (a scalar value), or having its shape as a
-  contiguous subset of the first tensor's shape. The starting of the mutually
-  equal shape is specified by the argument "axis", and if it is not set, suffix
-  matching is assumed. 1-dim expansion doesn't work yet.
+  tensor can either be of element size 1 (including a scalar tensor and any
+  tensor with rank equal to or smaller than the first tensor), or having its
+  shape as a contiguous subset of the first tensor's shape. The starting of the
+  mutually equal shape is specified by the argument "axis", and if it is not set,
+  suffix matching is assumed. 1-dim expansion doesn't work yet.
   
   For example, the following tensor shapes are supported (with broadcast=1):
   
-    shape(A) = (2, 3, 4, 5), shape(B) = (,), i.e. B is a scalar
+    shape(A) = (2, 3, 4, 5), shape(B) = (,), i.e. B is a scalar tensor
+    shape(A) = (2, 3, 4, 5), shape(B) = (1, 1), i.e. B is an 1-element tensor
```

Comment:
This change should be shown in the documentation of the following operators too, which also have broadcast attribute - AND, Equal, Xor, Greater, Lesser, and GEMM.

Reply:
Repeatedly putting this chunk in each relevant op seems cumbersome. Shall we put the description of our broadcasting rule as a separate doc, and then refer to it in each op? This can also help keeping the broadcast semantics consistent across all relevant ops.

Reply:
sounds good.

Reply:
Yeah, I did so in the other PR.

Reply:
@anirudhacharya do you want to approve this PR? We will solve the repeated broadcasting text in the other PR #907, otherwise we cannot check in the Gemm test cases.

Reply:
@houseroad approved.